backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestEvent.java

```diff
@@ -42,14 +44,26 @@
   @Column(columnDefinition = "uuid")
   private UUID priorCorrectedTestEventId; // used to chain events
 
+  private Boolean patientHasPriorTests;
```

Comment:
Honest question: what's the benefit of using the `Boolean` wrapper class instead of the `boolean` primitive? Is it because this value might be `null`?

Reply:
Yup, considering that existing testing events will all have `null` for `patientHasPriorTests` column.